README.md

```diff
@@ -16,10 +16,24 @@ Android Engineer Interview Questions
 ## Android
  - How does Garbage collection works?
  - What is a dangling pointer?
+ - Explain the terms reference counting, mark and sweep? 
  - Elaborate Memory Leak?
  - Explain fragment Lifecycle when it comes to ViewPager and sliding between different fragments.
  - Difference between FragmentStateAdapter and FragmentStatePagerAdapter.
- - 
+ - Difference between Serializable and Parcelable? What are the disadvantages of Serializable?
+ - How you could implement observable SharedPrefs or observable Databases i.e. Observe a certain key/table/query?
+ - How does layout inflation work from xml tags to view reference in memory?
+ - How would you go about implementing findViewById? Given 2 view references, how would you go about finding their ancestor view?
+   https://obvious.in/blog/deflating-the-layoutinflater-working-with-findviewbyid
+ - What is a thread, handler, looper and message queue?
+ - What are the different methods of concurrency on Android? Can you explain the difference between ExecutorService vs CachedThreadPool vs FixedThreadPool vs AsyncTasks vs HandlerThreads?
+ - How does ViewModel instance is provided to Activity, Fragment. How does ViewModelProviderStore decide when to retain the instance?
+  Ans : It uses <Key,Value> to hold onto viewmodel instance, these are removed when ViewModelStore#clear is called. 
+  Tip : Look at the accessor of this method to understand how it is retain across configuration changes
```

Comment:
We can skip adding the answer for now. I am planning a different approach to bring the Ans in for all the questions.